File: 782-jewels-and-stones/jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        std::set<char>Jset={};
        int count=0;
        for(int i=0;i<jewels.size();i++)
        {
            Jset.insert(jewels[i]);
        }
        for(int i=0;i<stones.size();i++)
        {
            if(Jset.find(stones[i])!=Jset.end())
            {
                count++;
            }
        }
        return count;
    }
};

File: 782-jewels-and-stones/jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        Jset=set()
        count=0
        for i in range(len(jewels)):
            Jset.add(jewels[i])
        for i in range(len(stones)):
            if((stones[i]) in Jset):
                count+=1
        return count
        